ludwig/datasets/__init__.py

```diff
@@ -81,16 +83,22 @@ def load_dataset_uris(
 
     Returns the input unmodified for any non-Ludwig datasets.
     """
-    dataset_out = dataset
-    training_set_out = training_set
-    validation_set_out = validation_set
-    test_set_out = test_set
+
+    if dataset is None and training_set is None:
+        raise ValueError("Neither dataset nor training_set has been provided. Please provide one of the two.")
```

Comment:
Are we checking anywhere when `dataset is None and training_set is None`?

Reply:
I this situation shouldn't come up because I think we raise an error elsewhere if this is true

Reply:
I would expect that the place where the error is raised elsewhere also raise this ValueError. Can you check?

Reply:
https://github.com/ludwig-ai/ludwig/blob/2772e9a36fffc1bc9f7d8403a0fbca1ca5b8258e/ludwig/data/preprocessing.py#L1082

Reply:
Sorry maybe I misunderstood what you were saying. Did you want me to do something with this ValueError in preprocessing.py?